buildSrc/src/main/kotlin/dev/nolij/zumegradle/JarCompressing.kt

```diff
@@ -34,7 +31,7 @@ enum class DeflateAlgorithm(val id: Int?) {
 	SEVENZIP(3),
 //	ZOPFLI(4), // too slow
 	;
-	
+
```

Comment:
Why is it removing empty line indents in Kotlin but adding them in Java? Please update `.editorconfig` so they match. I prefer indents on empty lines but prioritize consistency if you can have only one.